src/platform/intel/cavs/platform.c

```diff
@@ -111,7 +111,7 @@ static const struct sof_ipc_window sram_window
 			.type	= SOF_IPC_REGION_DEBUG,
 			.id	= 2,	/* map to host window 2 */
 			.flags	= 0, // TODO: set later
```

Comment:
@tlauda nice catch. Was there any side effect for size being wrong?

Reply:
@dbaluta It was caught by @zrombel, but I'm not sure whether there are any side effects.